docs/yaml.md

```diff
@@ -47,6 +47,10 @@ contain a kernel modules directory. `cmdline` specifies the kernel command line
 To override the names, you can specify the kernel image name with `binary: bzImage` and the tar image
 with `tar: kernel.tar` or the empty string or `none` if you do not want to use a tarball at all.
 
+Kernel packages may also contain a cpio archive containing CPU microcode which needs prepending to
+the initrd. To select this option, recommended when booting on bare metal, add `ucode: intel-ucode.cpio`
+to the kernel section.
```

Comment:
If I want to produce an image that can boot both Intel and AMD do I need to list them both (if/when AMD firmware is added if you haven't already) or is their a combined version?

Reply:
You'd need to produce a `cpio` archive containing both `GenuineIntel.bin` and `AuthenticAMD.bin`. Currently only Intel requires early boot ucode loading (I believe since Haswell) while AMD apparently can still use `/lib/firmware` style loading. I think if/when AMD requires early initrd loading we add `intel-ucode.cpio`, `amd-ucode.cpio`, and `ucode.cpio` to the kernel package.

Reply:
I haven't found the relevant bits of code/scripts for AMD but haven't looked very hard after I read on various sites it wasn't necessary. See:
- https://wiki.archlinux.org/index.php/microcode
- https://packages.debian.org/buster/amd64/amd64-microcode/filelist
I'm sure I saw something like this on AMD's website as well but can't find it anymore...